operations/workplace/people/hr-cadences.md

```diff
@@ -10,15 +10,74 @@ Listening sessions are 1x1 meeting with your HR partner. The intent of the HR Li
+When there is strong alignment with the Mission, [Mattermost Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles) and role expectations, individuals are able to self prioritize their work, [iterate](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) as needed, collaborate with impact and earn trust of their peers. 
+
+#### **Q9. My associates or fellow staff are committed to doing quality work.** 
+High performers are motivated to continue high performance when they feel their peers and colleagues are similarly committed. Similarly, high performers can be motivated to not continue high performance when they feel their peers or colleagues are not similarly committed. Maintaining high-trust in a remote environment requires intent from every team member at Mattermost. We can not physically “see” someone’s commitment to work, and micromanagement doesn’t work with our [leadership principles](hhttps://handbook.mattermost.com/company/about-mattermost#leadership-principles) or our culture of high-trust. To mitigate assumption-based thought, we want to focus on improving and communicating standards for asynchronous communications, transparent AORs and empathetic remote norms (e.g. response times, time zones) as part of Mattermost’s culture of high trust.  
+
+#### **Q10. I have a best friend at Mattermost.** 
+Socializing is physically difficult in a remote environment but not impossible. However, this factor does prioritize our focus to foster connections across Mattermost. We have a great team of staff and contributors around the world and the opportunity to create connections across teams and within our global community is a Mattermost differentiator. 
+
+The term “best friend” in this question is intended to mean quality of work relationships. When camaraderie exists within and across teams, issues are more likely to be solved with greater ease, clarity, speed and [trust](https://handbook.mattermost.com/company/about-mattermost#leadership-principles). When camaraderie is low, [high impact issues](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) or problems that require partnership are more likely to be avoided. 
+
+#### **Q11. In the last six months, someone at Mattermost has talked to me about my progress.** 
+Performance reviews are completed 1-2 times a year, depending on the team. Individuals expect performance feedback at more frequent intervals but may not always ask for feedback directly. Giving and receiving feedback are often avoided because the topic is discomforting. Fostering the value of [iteration](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) depends on the willingness to objectively hear and share feedback in order to grow individually, as a team and at Mattermost. 
```

Comment:
```suggestion
Performance reviews are completed once or twice a year, depending on the team's cadence. Individuals expect performance feedback at more frequent intervals but may not always ask for feedback directly. 

Giving and receiving feedback are often avoided because the topic is discomforting. Fostering the value of [iteration](https://docs.google.com/presentation/d/1BDSaeW-M92gth_NM1vI23dtbFLBsYVIk/edit#slide=id.g6f3b66b1fb_0_352) depends on the willingness to objectively hear and share feedback in order to grow individually, as a team and at Mattermost. 
```